contracts/src/LightClient.sol

```diff
@@ -182,27 +182,31 @@ contract LightClient is Initializable, OwnableUpgradeable, UUPSUpgradeable {
     /// @dev Initialization of contract variables happens in this method because the LightClient
     /// contract is upgradable and thus has its constructor method disabled.
     /// @param _genesis The initial state of the light client
-    /// @param _genesisStakeTableState The initial stake state of the light client
+    /// @param _genesisStakeTableState The initial stake table state of the light client
     /// @param _stateHistoryRetentionPeriod The maximum retention period (in seconds) for the state
     /// history
     function _initializeState(
         LightClientState memory _genesis,
         StakeTableState memory _genesisStakeTableState,
         uint32 _stateHistoryRetentionPeriod
     ) internal {
-        // stake table commitments and threshold cannot be zero, otherwise it's impossible to
-        // generate valid proof to move finalized state forward.
-        // Whereas blockCommRoot can be zero, if we use special value zero to denote empty tree.
-        // feeLedgerComm can be zero, if we optionally support fee ledger yet.
+        // The viewNum and blockHeight in the genesis state must be zero to indicate that this is
+        // the
```

Comment:
This paragraph is strangely formatted.

Reply:
https://github.com/EspressoSystems/espresso-sequencer/commit/58f43df708454d978a40e7614f154abd819ecdb4